Added Polymer logo to hero

yuinchien opened the pull request.

keanulee commented on app/index.html:
```diff
@@ -17,13 +17,31 @@
   </header>
 
   <style>
+    #polymer-logo {
```
Prefer class over id, even if only a single instance on page.

Why `<span>` with background-image instead of `<img>` tag? Prefer `<img>` unless there's a good reason, like CSS limitations.

Move this entire `<style>` tag above `<header>` so that all the elements it styles are below the `<style>` tag.

keanulee commented on app/index.html:
```diff
@@ -17,13 +17,31 @@
     <div class="wrapper">
-      <div class="hero-title">Polymer Project</div>
+      <div class="hero-title"><span id="polymer-logo"></span>Polymer Project</div>
+      <!-- <div class="hero-title">Polymer Project</div> -->
```
Remove comment.

frankiefu commented:
@yuinchien The logo looks good. One feedback from the team is we would also like the logo added before the "Polymer Project" title in the /blog page.

<img width="557" alt="screen shot 2018-12-05 at 12 02 49 pm" src="https://user-images.githubusercontent.com/116360/49540757-b5d87200-f885-11e8-8dd1-49909d9f6476.png">

yuinchien commented:
Will do that!

yuinchien commented:
@keanulee i updated the logo lockup styling & html markup to  <div id="logo-lockup" class="hero-title">Polymer Project</div>, and used pseudo element for the logo image. I used ID for logo-lockup because i wanna make sure it's only used once in landing page that's needed (only polymer-project & /blog). 

I've also updated the layout for the /blog page.

keanulee commented on app/blog/index.html:
```diff
@@ -16,24 +16,40 @@
+  <div class="wrapper">
+
+    {% for item in articles %}
+    <blog-tile style="display: block; border-top: 1px solid #FF4470; margin-bottom: 16px;">
```
`<blog-tile>` is not a custom element - use class name for styling instead, and move styles to templates/base-blog.html.

keanulee replied:
Correction: move styles to main.css

yuinchien replied:
sure.

keanulee commented on app/css/main.css:
```diff
@@ -90,6 +90,48 @@ pre {
+}
+
+
+/* blog styling */
```
Blog styles should be moved to templates/base-blog.html

keanulee replied:
Nevermind, templates/base-blog.html is for a single blog post. Disregard this comment.

yuinchien replied:
these styling are shared among base-blog.html & /blog, that's why it's moved to main.css

frankiefu commented:
@yuinchien One comment on the new UI: When the viewport width < 400px the project title gets cut off.
<img width="363" alt="screen shot 2018-12-10 at 12 33 12 pm" src="https://user-images.githubusercontent.com/116360/49759879-c7e85500-fc77-11e8-8e5c-4cd0bd88776d.png">

yuinchien commented:
@frankiefu this is the tricky case... i guess i can shrink down the logo-lockup even more for mobile screen.

keanulee commented:
Deployed to v
